symphony-bdk-core/src/test/java/com/symphony/bdk/core/util/PresentationMLParserTest.java

```diff
@@ -3,23 +3,36 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
 
 import com.symphony.bdk.core.service.message.exception.PresentationMLParserException;
 import com.symphony.bdk.core.service.message.util.PresentationMLParser;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class PresentationMLParserTest {
+
+  static Stream<Arguments> validPresentationMLs() {
+    return Stream.of(
+        arguments(
+            "<div data-format=\"PresentationML\" data-version=\"2.0\">\n<a href=\"http://www.symphony.com\">This is a link to Symphony's Website</a>\n</div>",
+            "This is a link to Symphony's Website"),
+        arguments("<div data-format=\"PresentationML\" data-version=\"2.0\"> <p>/test &lt;/messageML&gt;</p> </div>",
+            "/test </messageML>"),
+        arguments("<div data-format=\"PresentationML\" data-version=\"2.0\">Hello&#xA0;World</div>", "Hello World"));
```

Comment:
Please note that here I changed &NBSP; with its hexadecimal notation &#xA0; 
The test was failing because &NBSP; entity is not predefined in the doctype. (see https://stackoverflow.com/questions/13012327/error-parsing-page-xhtml-error-tracedline-42-the-entity-nbsp-was-referenc) 

Is it a breaking change or not?

Reply:
I wouldn't say so. To be sure, please run the same unit tests without changes

Reply:
To make it clear: 
Before my changes, when we put the presentationML with **_Hello&nbsp;World_** the returned content was **_HelloNBSPWorld_** . Now with my changes, the presentationML should have **_Hello&#xA0;World_** instead of **_HelloNBSPWorld_**, otherwise an error is thrown. 
The error and more explanations are in stackoverflow link in my first comment above.

Reply:
I think we added this test because the client could generate such content with &nbsp;
so we are not testing the same thing now are we?